File: Backend/src/main/java/cat/copernic/backendProjecte3/api/auth/dto/LoginResponse.java
package cat.copernic.backendProjecte3.api.auth.dto;

/**
 * Resposta del login.
 *
 * @param token Token de sessió generat pel servidor.
 * @param role Rol de l'usuari autenticat.
 */
public record LoginResponse(String token, String role) {
}

File: Mobile/app/src/main/java/cat/copernic/odecoches/data/remote/dto/LoginResponse.kt
package cat.copernic.odecoches.data.remote.dto

/**
 * Resposta del login.
 *
 * @param token Token de sessió generat pel servidor.
 * @param role Rol de l'usuari autenticat.
 */
/**
 * Resposta del backend amb la informació de la sessió autenticada.
 */
data class LoginResponse(
    val token: String,
    val role: String
)
